docs/guides/migrate-1.md

```diff
@@ -40,17 +57,27 @@ Update it to remove the `--develop` flag and add the `--static` flag instead:
 done-serve --static ...
 ```
 
-> *Note*: removing the `--develop` flag will disable live-reload. As we upgrade our application it will be in a broken state most of the time, so live-reload will not work anyways. When we've completed the upgrade we can switch back on `--develop`.
+> *Note*: removing the `--develop` flag will disable live-reload. As we upgrade our application, it will be in a broken state most of the time, so live-reload will not work anyway. When we’ve completed the upgrade, we can switch `--develop` back on.
 
 ## Front-end
 
 ### Upgrade to CanJS 3.0
 
-See the [CanJS 3 migration guide](http://canjs.com/doc/migrate-3.html) for instructions on how to migrate CanJS. Once you've made all necessary changes it is likely that your tests will still not pass. *Don't worry*! Some of the DoneJS dependencies you have installed still expect to use CanJS 2.3, once we've upgraded all of those the tests should pass again.
+See the [CanJS 3 migration guide](https://canjs.com/doc/migrate-3.html) for instructions on how to migrate CanJS. Once you’ve made all necessary changes, it is likely that your tests will still not pass. *Don’t worry*! Some of the DoneJS dependencies you have installed still expect to use CanJS 2.3; once we’ve upgraded all of those, the tests should pass again.
 
 ### Upgrade other DoneJS libraries
 
-The following packages should be updated to reflect the following semver ranges:
+Use npm to install the latest versions of the following packages:
+
+```shell
+npm install can-connect@1 --save
+npm install can-fixture@1 --save-dev
+npm install can-zone@0.6 --save
+npm install done-autorender@1 --save
+npm install done-component@1 --save
```

Comment:
I debated whether these should be listed as separate commands (as written above) or just two commands, one for `save` and the other for `save-dev` (e.g. `npm install can-connect@1 can-zone@0.6 done-autorender@1 done-component@1 --save`). Thoughts welcome!